backoffice/templates/backoffice/vehicles/details.html

```diff
@@ -0,0 +1,96 @@
+{% extends "backoffice_base.html" %}
+
+{% load static %}
+{% block scripts %}
+<script type="module" src="{% static "get-telemetry.js" %}" defer></script>
+{% endblock %}
+
+{% block heading %}<h2 class="bg-yellow-400 text-black font-bold text-xl px-6 py-2 rounded-md border-4 border-black tracking-widest shadow-md w-fit">{{ vehicle.registration }}</h2>{% endblock %}
+{% block buttons %}
+
+{% endblock %}
+
+{% block main %}
+
+<a class="hidden" href="{{ request.get_host }}" id="base_url"></a>
+<a href="{% url "get_telemetry" %}" class="hidden" id="telemetry_url"></a>
+<input type="hidden" id="vehicle_id" value="{{ vehicle.id }}">
+{% csrf_token %}
+
+<div class="m-8 flex flex-row gap-2">
+    <img src="{% if vehicle.picture %}{{ vehicle.picture.url }}{% endif %}""><div class="relative w-32 h-32">
+    <svg class="w-full h-full transform -rotate-90" viewBox="0 0 100 100">
+        <circle class="text-gray-200 stroke-current" cx="50" cy="50" r="40" stroke-width="10" fill="transparent" />
+{% if most_recent.soc.value > 75 %}
+        <circle class="text-green-500 stroke-current" cx="50" cy="50" r="40" stroke-width="10" fill="transparent"
+stroke-dasharray="251.2" stroke-dashoffset="{{ most_recent.soc_dial }}" stroke-linecap="round" />
+{% elif most_recent.soc.value > 50 %}
+        <circle class="text-amber-500 stroke-current" cx="50" cy="50" r="40" stroke-width="10" fill="transparent"
+stroke-dasharray="251.2" stroke-dashoffset="{{ most_recent.soc_dial }}" stroke-linecap="round" />
+{% else %}
+        <circle class="text-red-500 stroke-current" cx="50" cy="50" r="40" stroke-width="10" fill="transparent"
+stroke-dasharray="251.2" stroke-dashoffset="{{ most_recent.soc_dial }}" stroke-linecap="round" />
+{% endif %}
+    </svg>
+    <div class="absolute inset-0 flex items-center justify-center text-xl font-bold text-gray-800">
+      {{ most_recent.soc.value }}%
+    </div>
+  </div>
+</div>
+<h3 class="text-xl font-semibold text-gray-900 m-8">Previous, Current and Next Bookings for {{ vehicle.registration }}</h3>
+{% include "backoffice/bookings/bookings_list.html" with bookings=bookings %}
+
+<h3 class="text-xl font-semibold text-gray-900 m-8">Most Recent Telemetry Data</h3>
```

Comment:
This looks a bit odd as a narrow table half way down the page between two full-width ones. Maybe it could go at the top on larger screen sizes with a half/half horizontal split between it and the dials/images currently at the top? Not 100% sure it would look amazing, but think it's worth experimenting with...